drv/sprot-api/README.md

```diff
@@ -257,7 +257,6 @@ Message types include:
     to test the RoT's ability to exchange messages without error.
     The `SinkRsp` is just a header, no payload.
     On error, an `ErrorRsp` message is sent.
-  - _Unknown_ - 0xff A reserved, internal representation for an unsupported message type.
```

Comment:
It looks like `Unknown` still exists; why delete it here?

Reply:
`Unknown` has been removed as of https://github.com/oxidecomputer/hubris/pull/1261/commits/252293b67129f59566d41e5bf15ff991f9d1d32b